components/merino_client/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "merino_client"
+edition = "2021"
+version = "0.1.0"
+authors = ["contextual-services-team@mozilla.com"]
+license = "MPL-2.0"
+exclude = ["/android", "/ios"]
+
+[dependencies]
+anyhow = "1.0"
+error-support = { path = "../support/error" }
+log = "0.4"
+serde = "1"
+serde_derive = "1"
+serde_json = "1"
+thiserror = "1.0"
+uniffi = "^0.21"
+uniffi_macros = "^0.21"
+url = "2.1" # mozilla-central can't yet take 2.2 (see bug 1734538)
```

Comment:
It'll be a while before we can integrate Merino client into Desktop—or we might decide not to—but I copied this line from another component's `Cargo.toml` and kept it, because it's a good reminder just in case.

Reply:
We should be able to get [workspace dependencies](https://doc.rust-lang.org/nightly/cargo/reference/specifying-dependencies.html#inheriting-a-dependency-from-a-workspace) soon - we *almost* had it for UniFFI, but there was a problem vendoring into desktop so was backed out in 6ecde0b247d9c9b8129c680b454f13605c4349dc and I really can't remember whether that problem was uniffi specific or not, but @bendk said he thinks we should be able to get it sorted soon.

Reply:
Actually, I'm not so sure this can be sorted.  When our crates get vendored into moz-central, we don't have our workspace `Cargo.toml` anymore so this doesn't work.  I think this is fixable, but I've decided to just duplicate the dependencies for a while more and not try to fix it.